buyer-api/src/operations/addDataResponse.js

```diff
@@ -20,23 +20,24 @@ import { addProcessDataResponseJob } from '../queues/dataResponseQueue';
 export const addDataResponse = async (dataOrder, dataResponse) => {
   const { status: st, notariesAddresses } = dataOrder;
   if (st !== 'created') {
```

Comment:
It should still accept data responses when the offer is `closing`.
```suggestion
  if (st === 'creating' || st === 'closed') {
```